marketplace/deployer_util/config_helper.py

```diff
@@ -160,6 +160,13 @@ def validate(self):
               'schema v2. Images must be declared in the top level '
               'x-google-marketplace.images')
 
+    # Backward-incompatible validations that should only be enforced upon
```

Comment:
This is the intention of all `validate()` methods. This particular location for the comment is a bit misleading, since "here" does sound like "this line".

Maybe we should add this snippet to `Schema.validate()`, and make all other `validate()` methods reference it?

Reply:
Good call. Moved this comment to be part of the `Schema.validate()` docstring, and updated the `SchemaXServiceAccount.validate()` comment to reference it.